addok/config/default.py

```diff
@@ -11,6 +11,7 @@
     "documents": {
         "db": os.environ.get("REDIS_DB_DOCUMENTS") or 1,
     },
+    "ssl": os.environ.get("REDIS_SSL") == 'True',
```

Comment:
Boolean env vars are mostly 1/0. We should at least support 1,YES,TRUE (case insensitive).
WDYT?